### PR TITLE
Add admin name in /gtfo kick message

### PR DESCRIPTION
**Pull request by LegendIsAwesomes**
On the kick player area adds the sender (Admin) who banned the op.

It will clear up confusion when OPS do not know who banned them.

c3b09a64f6a5f3a7405c818a8e398045c7ce3ea9

### CODE REVIEW

**Comment by LegendIsAwesomes** [comment]
@AlexFreedomMod Yes, Say that i did /gtfo Alex (REASON)

without a name at the end it will automatically put the sender/admins name (In this case me)

**Comment by AlexFreedomMod** [comment]
[Fake Minecraft kick screen]

You have been banned, GTFO! Reason - (REASON) - @LegendIsAwesomes

**Comment by LegendIsAwesomes** [comment]
Yes my name would show at the end if i was the person who GTFO'd you @JeromSar This should work.

**Comment by LegendIsAwesomes** [comment]
@jeromsar tested it also tested my other request. Both work.

File: src/me/StevenLawson/TotalFreedomMod/Commands/Command_gtfo.java [hunk 1]
```diff
@@ -98,8 +98,7 @@ public boolean run(CommandSender sender, Player sender_p, Command cmd, String co
         TFM_BanManager.addUuidBan(new TFM_Ban(TFM_UuidManager.getUniqueId(player), player.getName(), sender.getName(), null, reason));
 
         // kick Player:
-        player.kickPlayer(ChatColor.RED + "GTFO" + (reason != null ? ("\nReason: " + ChatColor.YELLOW + reason) : ""));
-
+        player.kickPlayer(ChatColor.RED + "You have been banned, GTFO!" + (reason != null ? ("\nReason: " + ChatColor.YELLOW + reason) : "") + " - " + sender.getName());
```
Comment (JeromSar) [inline src/me/StevenLawson/TotalFreedomMod/Commands/Command_gtfo.java]: I prefer the first line to stay just "GTFO".

Please state the ban author on a new line, much like the reason.

**Comment by LegendIsAwesomes** [comment]
@JeromSar What exactly needs changing. I have used this on numerous mods i have made. Works perfectly

**Comment by Scuph** [comment]
Can this be merged please, it's fine...

@JeromSar

**Comment by LegendIsAwesomes** [comment]
@scuph did you use this?. I use this on multiple servers I develop in. It works fine

File: src/me/StevenLawson/TotalFreedomMod/Commands/Command_gtfo.java [hunk 2]
```diff
@@ -98,8 +98,7 @@ public boolean run(CommandSender sender, Player sender_p, Command cmd, String co
         TFM_BanManager.addUuidBan(new TFM_Ban(TFM_UuidManager.getUniqueId(player), player.getName(), sender.getName(), null, reason));
 
         // kick Player:
-        player.kickPlayer(ChatColor.RED + "GTFO" + (reason != null ? ("\nReason: " + ChatColor.YELLOW + reason) : ""));
-
+        player.kickPlayer(ChatColor.RED + "GTFO!" + (reason != null ? ("\nReason: " + ChatColor.YELLOW + reason) : "") + " \nBanned by: " + sender.getName());
```
Comment (JeromSar) [inline src/me/StevenLawson/TotalFreedomMod/Commands/Command_gtfo.java]: Trailing whitespace.

**Comment by JeromSar** [comment]
@LegendIsAwesomes Thanks for contributing! Please make the necessary changes. :)

**Comment by LegendIsAwesomes** [comment]
@JeromSar what specifically needs changing? ChatColor.RED instead of yellow for the banners name

**Comment by Wild1145** [comment]
Can you also change the title of the PR please. "Update command_gtfo.java" is incredibly vague...

**Comment by JeromSar** [comment]
@LegendIsAwesomes There's a trailing whitespace in the message.

**Comment by LegendIsAwesomes** [comment]
@JeromSar  https://github.com/LegendIsAwesomes/TotalFreedomMod-1/commit/f5860d03568e90536c27d80034fca3a8bf0701e5 (I don't exactly know the definiton of trailing whitespace, So i took a guess, Maybe im right

**Comment by JeromSar** [comment]
@LegendIsAwesomes On this PR, it's right before `\nBanned By`. The last space on that line is never shown, since it's just before a linebreak.

**Comment by LegendIsAwesomes** [comment]
@JeromSar  so where is says  " \nBanned by: " + sender.getName()); the space between the ending " and the : is not needed?

**Comment by JeromSar** [comment]
@LegendIsAwesomes No, I didn't say that. Read again what I said.

**Comment by SlicedCaek** [comment]
Jerom said, On this PR, it's right before ``` \nBanned ``` By. The last space on that line is never shown, since it's just before a linebreak.

You made ``` " \nBanned" ``` and he saying to remove the space before the ``` \nBanned ``` so basically its ``` "\nBanned" ```

Correct me if i am wrong :+1:

**Comment by JeromSar** [comment]
@LegendIsAwesomes Looks good, merging soon. :)

**Comment by LegendIsAwesomes** [comment]
@JeromSar Thanks. Finally got it right xD

**Comment by LegendIsAwesomes** [comment]
@JeromSar what's wrong with this?

**Comment by LegendIsAwesomes** [comment]
I accepted it

**Comment by Wild1145** [comment]
@JeromSar Re-Open?

**Pull request closed without merging**